src/apps/possumwood/node_menu.cpp

```diff
@@ -0,0 +1,168 @@
+#include "node_menu.h"
+
+#include <boost/algorithm/string/classification.hpp>
+#include <boost/algorithm/string/join.hpp>
+#include <boost/algorithm/string/split.hpp>
+#include <boost/filesystem.hpp>
+
+#include <actions/actions.h>
+#include <actions/node_data.h>
+#include <possumwood_sdk/app.h>
+
+#include "searchable_menu.h"
+
+namespace {
+
+QAction* makeAction(QString title, std::function<void()> fn, QWidget* parent) {
+	QAction* result = new QAction(title, parent);
+	QObject::connect(result, &QAction::triggered, [fn](bool) { fn(); });
+	return result;
+}
+
+}  // namespace
+
+NodeMenu::NodeMenu(Adaptor* a) : m_adaptor(a) {
+	m_newNodeMenu = std::unique_ptr<QMenu>(new SearchableMenu("Create\tTab"));
```

Comment:
Codacy found an issue: [Variable 'm_newNodeMenu' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/manual/martin.prazak/possumwood/pullRequest?prid=6093062)